rpc/src/rpc.rs

```diff
@@ -2212,10 +2212,18 @@ impl JsonRpcRequestProcessor {
     fn get_recent_prioritization_fees(
         &self,
         pubkeys: Vec<Pubkey>,
+        percentile: Option<u16>,
     ) -> Result<Vec<RpcPrioritizationFee>> {
-        Ok(self
-            .prioritization_fee_cache
-            .get_prioritization_fees(&pubkeys)
+        let cache = match percentile {
+            Some(percentile) => self
+                .prioritization_fee_cache
+                .get_prioritization_fees2(&pubkeys, percentile),
+            None => self
+                .prioritization_fee_cache
+                .get_prioritization_fees(&pubkeys),
+        };
+
+        Ok(cache
```

Comment:
```suggestion
        Ok(self
            .prioritization_fee_cache
            .get_prioritization_fees(&pubkeys, percentile.unwrap_or(0))
```
As per my comment on `get_prioritization_fees2`